src/Components/Header.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import MicIcon from '@mui/icons-material/Mic';
+import SettingsIcon from '@mui/icons-material/Settings';
+import './Style/header.css';
```

Comment:
- [ ] When I run npm app I see error logs: 
<img width="1378" alt="Screenshot 2022-10-13 at 20 59 01" src="https://user-images.githubusercontent.com/77038610/195671325-48c1138c-dabb-4669-ab72-3e5023955611.png">
The reason of some errors is that you are referencing Styles folder with capital letter, but in fact it is written with lowercase letters
`import './Style/header.css' `=> import './style/header.css' 

Please update all components.